09_unique.rb

```diff
@@ -17,7 +17,37 @@
 # Check your solution by running the tests:
 # ruby tests/09_unique_test.rb
 #
+#
+#array style
+def unique (mylist)
+clean = []
+duplicate = []
+  mylist.each do |value|
+    if !clean.include?(value)
+        clean.push(value)
+    else
+      duplicate.push(value)
+    end
+  end
+  return clean
+end
+
+#hash style
+def unique (mylist)
+  clean = {}
+  duplicate = []
+  mylist.each do |value|
+    if value != true
+      clean[value] = value
+    else
+      duplicate.push(value)
+    end
+  end
+   clean.keys
```

Comment:
Awesome solution using `.keys`!

Reply:
if I didnt use that it just returned {1 => , etc and wouldnt pass test